Makefile

```diff
@@ -36,5 +36,10 @@ integration-test:
 	test/integration/test_serialization.py test/integration/test_ibm_job_attributes.py test/integration/test_basic_server_paths.py \
   test/integration/test_ibm_integration.py test/integration/test_ibm_job.py test/integration/test_ibm_qasm_simulator.py \
   test/integration/test_proxies.py test/integration/test_composite_job.py
+
+e2e-test:
+	# TODO: enable test_tutorials 
+	# python -m unittest discover --verbose --top-level-directory . --start-directory test/e2e
+
```

Comment:
@daka1510 I moved the tests to a e2e folder as highlighted above - this is commented out for now as the tutorials still need to be updated